frontend/src/styles/globals.scss

```diff
@@ -35,12 +34,23 @@ body,
   // empty space at the bottom of the page, below the footer.
   // Since the user can scroll in `.non-header-wrapper`, everything outside that
   // that still overflows these elements can safely be hidden:
-  overflow: hidden;
+  // overflow: hidden;
 }
 
 html {
   scroll-behavior: smooth;
   scroll-padding-top: $layout-xl;
+  font-family: $font-stack-base;
+  height: auto;
```

Comment:
Hmm crap. Unfortunately it's been a while since I touched this, so I don't have top of mind why this is happening, but it appears that without `height: 100%`, the mobile menu is no longer sticky:

[Screencast from 2023-01-12 11-14-05.webm](https://user-images.githubusercontent.com/4251/212040074-fd3100d5-2255-4712-b7d1-27cf38fb638f.webm)